go-controller/pkg/util/net.go

```diff
@@ -317,11 +317,10 @@ func IPsToNetworkIPs(ips ...*net.IPNet) []*net.IPNet {
 	return nets
 }
 
-// StringsToIPs takes a slice of strings and returns a slice of net.IPs
-func StringsToIPs(ips []string) []net.IP {
-	s := make([]net.IP, len(ips))
-	for i := range ips {
-		s[i] = net.ParseIP(ips[i])
+func IPNetsToString(ips []*net.IPNet) []string {
```

Comment:
Oh this is using the string method of the IP in the IPNet instead of the IPNet strig method itself.
Better name couild be `IPNetsIPToString`?

Reply:
yeah, it is IPNet.IP.String(). will rename this.